notebooks/working/reproducing_deconfounder_results.ipynb

```diff
@@ -0,0 +1,1680 @@
+       "</tr>\n",
+       "</table><br/><br/>Warnings:<br/>[1] Standard Errors assume that the covariance matrix of the errors is correctly specified."
+      ],
+      "text/plain": [
+       "<class 'statsmodels.iolib.summary.Summary'>\n",
+       "\"\"\"\n",
+       "                                 OLS Regression Results                                \n",
+       "=======================================================================================\n",
+       "Dep. Variable:                      y   R-squared (uncentered):                   0.002\n",
+       "Model:                            OLS   Adj. R-squared (uncentered):              0.002\n",
+       "Method:                 Least Squares   F-statistic:                              18.85\n",
+       "Date:                Wed, 29 Apr 2020   Prob (F-statistic):                    3.33e-12\n",
+       "Time:                        17:23:54   Log-Likelihood:                         -50073.\n",
+       "No. Observations:               23961   AIC:                                  1.002e+05\n",
+       "Df Residuals:                   23958   BIC:                                  1.002e+05\n",
+       "Df Model:                           3                                                  \n",
+       "Covariance Type:            nonrobust                                                  \n",
+       "==============================================================================\n",
+       "                 coef    std err          t      P>|t|      [0.025      0.975]\n",
+       "------------------------------------------------------------------------------\n",
+       "TOTALEXP      -0.0958      0.013     -7.435      0.000      -0.121      -0.071\n",
+       "marital       -0.0103      0.014     -0.765      0.445      -0.037       0.016\n",
+       "LASTAGE        0.0078      0.014      0.567      0.571      -0.019       0.035\n",
+       "==============================================================================\n",
+       "Omnibus:                     4944.725   Durbin-Watson:                   2.004\n",
+       "Prob(Omnibus):                  0.000   Jarque-Bera (JB):           149944.673\n",
+       "Skew:                          -0.277   Prob(JB):                         0.00\n",
+       "Kurtosis:                      15.243   Cond. No.                         1.52\n",
+       "==============================================================================\n",
+       "\n",
+       "Warnings:\n",
+       "[1] Standard Errors assume that the covariance matrix of the errors is correctly specified.\n",
+       "\"\"\""
+      ]
+     },
+     "execution_count": 12,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "mod = smf.ols('y ~ -1 + TOTALEXP + marital + LASTAGE', data = df)\n",
+    "results = mod.fit()\n",
+    "results.summary()"
```

Comment:
The results are really wild. You simulate `beta_exp = np.random.normal(loc=0, scale = 1, size = df.shape[0])`, but yet your OLS estimate for `beta_exp`, averaged over all individuals is `-0.0958` and OLS thinks it's significantly different from zero? Geez. This is a good reminder of why we should always be wary of one's model results. It might be good to mention this in a markdown cell beneath the results table!